### PR TITLE
Testbranch

### PR DESCRIPTION
**Pull request by hardik-kanjariya-b**

### CODE REVIEW

File: 17thjan/Task_1.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	puts star=star+"*"
+end
+
+=end
+
+
+#second way
+
+(1..5).each do |i|
+if(i==1)
```
Comment (hardik-kanjariya-b) [inline 17thjan/Task_1.rb:18]: Can you please explain of using if condition here

File: 17thjan/Task_4.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
+5.downto(1).each do |i|
```
Comment (hardik-kanjariya-b) [inline 17thjan/Task_4.rb:2]: This logic created 10 Stars in the first and last line, Please check your logic and modify accordingly

File: 17thjan/Task_5.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+puts "Enter Number:"
```
Comment (hardik-kanjariya-b) [inline 17thjan/Task_5.rb:1]: This logic works fine.. But You still need to reduce loop iterations

File: 17thjan/Task_1.rb (hunk 1 above)
Comment (khushali25) [reply 17thjan/Task_1.rb:18]: Yes sir i checked it if condition is not required 
 puts "*"*i works

Comment (hardik-kanjariya-b) [reply 17thjan/Task_1.rb:18]: resolve this, and push code again

Comment (khushali25) [reply 17thjan/Task_1.rb:18]: okay

**Pull request merged**